packages/docs-components/src/knobs/KnobInspector.tsx

```diff
@@ -46,30 +47,37 @@ const getKnobComponents = (
   return components
 }
 
-const KnobInspector: React.FunctionComponent = () => {
+type KnobInspectorProps = {
+  children?: (children: React.ReactElement) => React.ReactElement
+}
+
+const KnobInspector: React.FunctionComponent<KnobInspectorProps> = props => {
```

Comment:
We should decide whether we prefer `FC` or `FunctionComponent` and be consistent with it

Reply:
As I remember we agreed on `FunctionComponent`, @kuzhelov ?